### PR TITLE
enable caching for local centaur

### PR DESCRIPTION
**Pull request by ruchim**

### CODE REVIEW

**Comment by ruchim** [comment]
TRAVIS IS GREEEEN!

File: src/bin/travis/resources/centaur.wdl [hunk 1]
```diff
@@ -28,4 +29,4 @@ task centaur {
 }
 workflow centaur {
     call centaur
-}
+}
```
Comment (geoffjentry) [inline src/bin/travis/resources/centaur.wdl]: revert

File: src/bin/travis/resources/centaur.wdl [hunk 2]
```diff
@@ -11,6 +11,7 @@ task centaur {
         export SBT_OPTS=-Dsbt.ivy.home=/cromwell_root/tmp/.ivy2
         git clone https://github.com/broadinstitute/centaur.git
         cd centaur
+        git checkout rm_convertToStandardSpec
```
Comment (geoffjentry) [inline src/bin/travis/resources/centaur.wdl]: I'm assuming this is the case but just double checking that this is temporary

File: src/bin/travis/resources/local_centaur.conf [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  }
+  client {
+    request-timeout = 40s
+    connecting-timeout = 40s
+  }
+}
+
+call-caching {
+  enabled = true
+}
```
Comment (geoffjentry) [inline src/bin/travis/resources/local_centaur.conf]: \n

File: src/bin/travis/testCentaurLocal.sh [hunk 4]
```diff
@@ -31,6 +31,7 @@ set -e
 
 sbt assembly
 CROMWELL_JAR=$(find "$(pwd)/target/scala-2.11" -name "cromwell-*.jar")
+LOCAL_CONF="$(pwd)/src/bin/travis/resources/local_centaur.conf"
```
Comment (geoffjentry) [inline src/bin/travis/testCentaurLocal.sh]: should this file be checked in?

**Comment by geoffjentry** [comment]
👍

File: src/bin/travis/resources/centaur.wdl (hunk 2 above)
Comment (ruchim) [inline src/bin/travis/resources/centaur.wdl]: 'tis certainly temporary

**Comment by kshakir** [comment]
👍

**Pull request merged**